dodola/tests/test_services.py

```diff
@@ -636,6 +610,131 @@ def test_aiqpd_train(tmpdir, monkeypatch):
     assert aiqpd_model.af.shape == af_expected_shape
 
 
+def test_aiqpd_integration(tmpdir, monkeypatch):
+    """Tests that the shape of adjustment factors matches the expected shape"""
```

Comment:
I think this docstr needs to be updated if you don't mind.

Reply:
Thanks for noticing this! Updated.